lib/ex_doc/language/elixir.ex

```diff
@@ -559,6 +559,14 @@ defmodule ExDoc.Language.Elixir do
   end
 
   defp url(string, mode, config) do
+    if Enum.any?(config.skip_code_autolink_to, &(&1 == string)) do
```

Comment:
Should we allow regexes here as well?

Reply:
We can add if necessary later on? Unless you have an immediate need for it?

Reply:
I deprecated a module, so I'd like to do `~r/MyMod(\..*)?/` basically instead of having to list the references to the module alone and then to all the functions I mention. Unless I'm reading the feature wrong 😉

Reply:
not sure if this would ever come up but the above regex will also catch `MyNamespace.MyMod`. So maybe it should be: `~r/^MyMod(\..*)?/`. but then we'd still warn on callbacks and types so I suppose `~r/^([bt]:)?MyMod(\..*)?/`.

I think we can add the regex support but it needs to be used carefully. but yeah I'd rather have users spell out things explicitly to keep the code explicit too. how annoying is this gonna be for you? is this for a changelog? If so consider just cutting off old changelog entries, e.g. write:

The CHANGELOG for v1.15 and previous releases can be found [in the v1.15] branch

or something like that.

Reply:
Maybe we just make it a function and allow you to pass anything?

Reply:
Nice, +1 from me.

Reply:
I will merge this and we can make both `skip_` options accept a function too.